docs/mp/jaxrs/12_jaxrs-client.adoc

```diff
@@ -0,0 +1,165 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Jakarta REST (JAX-RS) Client
+:h1Prefix: MP
+:feature-name: Jakarta REST Client
+:description: Jakarta REST (JAX-RS) Client
+:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
+:rest-client: ../restclient/09_rest-client.adoc
+:jaxrs-spec: https://jakarta.ee/specifications/restful-ws/3.0/jakarta-restful-ws-spec-3.0.html
+:jaxrs-spec-client: https://jakarta.ee/specifications/restful-ws/3.0/jakarta-restful-ws-spec-3.0.html#client_api
+:jaxrs-javadocs: https://jakarta.ee/specifications/restful-ws/3.0/apidocs/
+= :keywords: helidon, rest, jax-rs, client, microprofile, micro-profile
+
+== ToC
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<API, API>>
+- <<Configuration, Configuration>>
+- <<Examples, Examples>>
+- <<Additional Information, Additional Information>>
+- <<Reference, Reference>>
+
+== Overview
+
+The Jakarta REST Client defines a programmatic API to access
+REST resources. This API sits at a higher level than traditional HTTP client APIs and
+provides full integration with server-side API concepts like providers. It differs
+from the link:{rest-client}[Rest Client API] in that it does not support
+annotations or proxies, but instead uses builders and a fluent API to
+create and execute requests.
+
+include::{common-deps-page-prefix-inc}[tag=maven-dependency]
+[source,xml]
+----
+     <dependency>
+         <groupId>io.helidon.jersey</groupId>
+         <artifactId>helidon-jersey-client</artifactId>
+     </dependency>
+----
+
+== API
+
+Bootstrapping the API is done by obtaining an instance of `Client`. A single instance of this
+class can be used to create multiple service requests that share the same basic
+configuration, e.g., the same set of _providers_. More precisely, from a `Client` we
+can create multiple `WebTarget` 's, and in turn, from each `WebTarget` we can
```

Comment:
WebTargets Invocations -- no need for 's.

Reply:
WebTarget is a class name (WebTargets is not), hence the apostrophe and the backquotes.

Reply:
Dropped the apostrophe. I don't think the rendering is as clear with the quotes, but at least correct.